listmonk@.service

```diff
@@ -8,12 +11,23 @@ After=postgresql.service
 
 [Service]
 Type=simple
+PermissionsStartOnly=true
 EnvironmentFile=-/etc/default/listmonk
 EnvironmentFile=-/etc/default/listmonk-%i
-ExecStartPre=/usr/bin/mkdir -p "${HOME}/uploads"
+ExecStartPre=/usr/bin/mkdir -p "/etc/listmonk/uploads"
```

Comment:
This shouldn't default to `/etc/listmonk` as a) the app should be run as a non-root user and hence, permission to write to /etc/ is unlikely b) arbitrary user files/content should not go in `/etc/*`.

Reply:
Where should they go, given that `${HOME}` is incompatible with `ProtectHome=True`?

Reply:
Did a bit of Googling and looks like `BindPaths, ReadWritePaths` are the way to go. I am not familiar with it, so will have to experiment.